aspnetcore/host-and-deploy/windows-service/samples/2.x/AspNetCoreService/README.md

```diff
@@ -1,3 +1,103 @@
 # ASP.NET Core Windows Service Sample (Framework-dependent Deployment)
 
 This sample shows how to host an ASP.NET Core app as a Windows Service without using IIS. This sample demonstrates the scenario described in [Host an ASP.NET Core app in a Windows Service](https://docs.microsoft.com/aspnet/core/host-and-deploy/windows-service).
+
+## Logging level
+
+For demonstration and testing purposes, the sample app's Production environment settings file sets the logging level to `Information`. In production app, the value is typically set to `Error`. For more information, see [Windows Eventlog Provider](https://docs.microsoft.com/aspnet/core/fundamentals/logging/index#windows-eventlog-provider).
+
+## Publish the sample app
+
+To publish the sample app using command-line interface (CLI) tools, execute the [dotnet publish](https://docs.microsoft.com/dotnet/core/tools/dotnet-publish) command in a command shell from the project folder. The default project configuration is Release but can be changed by passing the [-c|--configuration](https://docs.microsoft.com/dotnet/core/tools/dotnet-publish#options) option to the command. Use the [-o|--output](https://docs.microsoft.com/dotnet/core/tools/dotnet-publish#options) option with a path to publish to a folder (the folder is created if it doesn't exist when the command is executed).
+
+## PowerShell
+
+Execute PowerShell commands from an administrative PowerShell 6 command shell.
+
+## Create a user account
+
+Create a user account with the name `ServiceUser`.
+
+On Windows 10 October 2018 Update (version 1809/build 10.0.17763) or later:
+
+```PowerShell
+New-LocalUser -Name ServiceUser
+```
+
+On Windows OS earlier than the Windows 10 October 2018 Update (version 1809/build 10.0.17763):
+
+```console
+powershell -Command "New-LocalUser -Name ServiceUser"
+```
+
+Provide a [strong password](https://docs.microsoft.com/windows/security/threat-protection/security-policy-settings/password-must-meet-complexity-requirements) when prompted.
+
+Unless the `-AccountExpires` parameter is supplied to the [New-LocalUser](https://docs.microsoft.com/powershell/module/microsoft.powershell.localaccounts/new-localuser) cmdlet with an expiration <xref:System.DateTime>, the account doesn't expire.
+
+## Provide the service user account with Log on as a service rights
+
+1. Open the Local Security Policy editor by running *secpool.msc*.
```

Comment:
Link back to specific sections of the doc rather than duplicating content? These instructions aren't specific to this sample.